cmd/action_config.go

```diff
@@ -18,7 +19,12 @@ func actionConfig(ctx *cli.Context) error {
 		log.Error("failed to load config", "err", err)
 		return err
 	}
-	s, _ := json.MarshalIndent(cfg, "", "\t")
-	fmt.Print(string(s))
+	if ctx.Bool(JsonFlag.Name) {
+		s, _ := json.MarshalIndent(cfg, "", "\t")
+		fmt.Print(string(s))
+	} else {
+		s, _ := toml.Marshal(cfg)
+		fmt.Print(string(s))
+	}
```

Comment:
Because these are distinct code paths let's add a test for both. It's fine if the test is just a copy paste with 1 line changed